tools/make-tarball

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+# shellcheck disable=2015,2039,2162
+
+TEMP_D=""
+cleanup() {
+    [ -z "$TEMP_D" ] || rm -Rf "${TEMP_D}"
+}
+stderr() { echo "$@" 1>&2; }
+fail() { [ $# -eq 0 ] || stderr "$@"; exit 1; }
+trap cleanup EXIT
+
+Usage() {
+    cat <<EOF
+Usage: ${0##*/} [revision]
+  create a tarball of revision (default HEAD)
+
+  options:
+    -h | --help             print usage
+         --long             Use git describe --long for versioning
+    -o | --output FILE      write to file
+         --version VERSION  Set the version used in the tarball. Default
+                            value is determined with 'git describe'.
+EOF
+}
+
+sopts="ho:v"
```

Comment:
This 'v' seems wrong?  Did you mean it as an alias for 'version'?  But if so then you wanted v: right?

Reply:
Copy pasta error. Normally it’s there for verbose. I’ll fix

Reply:
done